includes/classes/class-woo-civi-helper.php

```diff
@@ -450,7 +450,7 @@ public function get_price_sets_populated() {
 					}
 
 					// Add Tax data if necessary.
-					if ( $tax_enabled && empty( $tax_rates ) && array_key_exists( $price_field_value['financial_type_id'], $tax_rates ) ) {
```

Comment:
@rbaugh It looks to me like this is just a bug in the existing code and it should read:

```php
if ( $tax_enabled && ! empty( $tax_rates ) && array_key_exists( $price_field_value['financial_type_id'], $tax_rates ) ) {
```

This would trap returns of type boolean `false` and an empty array without the need to check if it's an array.

Reply:
That should work. Overall this was causing the issue in #37 as it was failing when trying to use it as an array when it was a bool.